### PR TITLE
updated seeder

### PR DESCRIPTION
**Pull request by jatnerubia**
Ticket ID: [Create seeder for Email Template - Create Evaluation](https://github.com/orgs/nerubia/projects/18/views/2?pane=issue&itemId=43991265)

### CODE REVIEW

**Review by Dev-Nino — APPROVED** [review]

File: src/seeders/seed.ts [hunk 1]
```diff
@@ -148,13 +105,76 @@ export const createEvaluationResults = async () => {
...
+  })
+  if (emailTemplate === null) {
+    await prisma.email_templates.create({
+      data: [
+        {
+          name: "Create Evaluation Administration Template",
+          template_type: "Create Evaluation",
+          is_default: true,
+          subject: "Request for Evaluation",
+          content: `Dear Evaluator,\n\nWe kindly request your participation in the evaluating the following during this period: {{evaluation_period}}. Please follow this {{link}} to access {{evaluation_name}}.\n\nThe deadline for completing these evaluations is on {{eval_schedule_end_date}}. If you encounter any technical issues or have questions regarding the process, please feel free to reach out to any of the HR team members.\n\nThanks and Best Regards,\nKH360 Team`,
```
Comment (emmaacha) [inline src/seeders/seed.ts:160]: Grammatical error?

Comment (emmaacha) [inline src/seeders/seed.ts:160]: Grammatical error?

**Review by emmaacha — CHANGES_REQUESTED** [review]

File: src/seeders/seed.ts (hunk 1 above)
Comment (jatnerubia) [reply src/seeders/seed.ts:160]: the content? 🤔 @emmaacha

Comment (emmaacha) [reply src/seeders/seed.ts:160]: Yes po kuys @jatnerubia this part: `"We kindly request your participation in the evaluating the following..."`. I think it should either be `"in evaluating the following"` or `"in the evaluation of the following"`

Comment (jatnerubia) [reply src/seeders/seed.ts:160]: ohh, ask shei about this

Comment (emmaacha) [reply src/seeders/seed.ts:160]: This is the final content kuys @jatnerubia according to ate Shei 😁
```
Dear Evaluator,

We kindly request your participation in evaluating the following during this period: {{evaluation_period}}.
Please follow this {{link}} to access {{evaluation_name}}.

The deadline for completing these evaluations is on {{eval_schedule_end_date}}.
If you encounter any technical issues or have questions regarding the process, please feel free to reach out to any of the HR team members.

Thanks and Best Regards,
KH360 Team
```

**Comment by jatnerubia** [comment]
@emmaacha done ^^

File: src/seeders/seed.ts [hunk 2]
```diff
@@ -119,21 +90,7 @@ const createUserDetails = async () => {
...
 
-export const createEmailTemplates = async () => {
-  await prisma.email_templates.createMany({
-    data: [
-      {
-        name: "Create Evaluation Administration Template",
-        template_type: "Create Evaluation",
-        is_default: true,
-        subject: "Request for Evaluation",
-        content: `Dear Evaluator,\n\nWe kindly request your participation in the evaluating the following during this period: {{evaluation_period}}. Please follow this {{link}} to access {{evaluation_name}}.\n\nThe deadline for completing these evaluations is on {{eval_schedule_end_date}}. If you encounter any technical issues or have questions regarding the process, please feel free to reach out to any of the HR team members.\n\nThanks and Best Regards,\nKH360 Team`,
```
Comment (emmaacha) [inline src/seeders/seed.ts:130]: New changes from ate Sheila pala kuys, will update here once it's available, thanks!

**Comment by jatnerubia** [comment]
@emmaacha done ^^

**Review by emmaacha — APPROVED** [review]
LGTM! 🚀

**Pull request merged**